pymc3/aesaraf.py

```diff
@@ -156,6 +156,10 @@ def change_rv_size(
             size = rv_node.op._infer_shape(size, dist_params)
         new_size = tuple(np.atleast_1d(new_size)) + tuple(size)
 
+    # Make sure the new size is a tensor. This helps to not unnecessarily pick
```

Comment:
```suggestion
    # Make sure the new size is an int64 tensor. This helps to not unnecessarily pick
```

Reply:
Should we also link to the issue here?

Reply:
It's discoverable through the commits.
But on #4696 we'll make a few more robustness changes to `change_rv_size`. I can link the issue when rebasing the `reintro_shape` branch.